bluto/doc.go

```diff
@@ -0,0 +1,6 @@
+/*Package bluto is connection pool wraper which can borrow redis connection from pool and return them when finished.
+Pooler manage and reuse existing conection and provide advanced config options for handling timeouts.  
```

Comment:
Pooler is no more. Use Bluto.

Typo:
manage => manages
reuse => reuses
connection => connections
for handling timeouts. => e.g. for handling timeouts.